sql/001-create-table-and-load-data.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS names;
+
+CREATE TABLE names (
+ id int unsigned AUTO_INCREMENT,
+ name VARCHAR(20) NOT NULL,
+ PRIMARY KEY(id)
+);
+
+INSERT INTO names (id,name) VALUES (1,"鈴木");
```

Comment:
```suggestion
INSERT INTO names (id, name) VALUES (1, "鈴木");
```
細かいですが、sqlもフォーマットしたい。

Reply:
承知しました！
修正します！